<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseDialog</class>
 <widget class="QDialog" name="CourseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>课程编辑</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basicInfoTab">
      <attribute name="title">
       <string>基本信息</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_courseId">
         <property name="text">
          <string>课程ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_courseId"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_courseName">
         <property name="text">
          <string>课程名称:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_courseName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_semester">
         <property name="text">
          <string>学期:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_semester"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_courseType">
         <property name="text">
          <string>课程类型:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_courseType"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_credits">
         <property name="text">
          <string>学分:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_credits">
         <property name="minimum">
          <double>0.500000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="classesTab">
      <attribute name="title">
       <string>教学班</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget_classes">
         <property name="columnCount">
          <number>3</number>
         </property>
         <column>
          <property name="text">
           <string>教师</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上课时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>操作</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_addClass">
         <property name="text">
          <string>添加教学班</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="prerequisitesTab">
      <attribute name="title">
       <string>前置课程</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="listWidget_prerequisites"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_addPrerequisite">
           <property name="text">
            <string>添加前置课程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_removePrerequisite">
           <property name="text">
            <string>移除前置课程</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CourseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
